lib/ui/geometry.dart

```diff
@@ -1237,7 +1237,15 @@ class RRect {
        assert(brRadiusX != null),
        assert(brRadiusY != null),
        assert(blRadiusX != null),
-       assert(blRadiusY != null);
+       assert(blRadiusY != null),
+       assert(tlRadiusX >= 0),
```

Comment:
Maybe mention this in the docs for the public constructors that feed into this?

Reply:
Good point.